File: file_c++/loop3.c
#include <stdio.h>
int n = 0;
int angka = 0;
int terkecil;

int main(){
    printf("Masukkan angka ke-1 : ");
    scanf("%d",&terkecil);
    for(int i = 2; i <= 10; i++){
        printf("Masukkan angka ke-%d : ",i);
        scanf("%d",&angka);
        if(angka <= terkecil){
            terkecil = angka;
        }
    }
    printf("angka terkecil adalah %d",terkecil);
}

File: file_py/loop3.py

terkecil = int(input("Masukkan angka ke-1 : "))
for i in range (1,11):
    n = int(input(f"Masukkan angka ke-{i} : "))
    if(n <= terkecil):
        terkecil = n
        
print(f"angka terkecil adalah : {terkecil}")